tests/integration/example_test_env_vars.yaml

```diff
@@ -11,3 +11,15 @@ post:
   ref: e1be527f39ec31323f0454f7d1422c6260b00580
   # Package managers
   pkg_managers: ["gomod"]
+# Test data for the Valid data in request test
+get:
+  repo: https://github.com/release-engineering/retrodep.git
+  ref: e1be527f39ec31323f0454f7d1422c6260b00580
+  pkg_managers: ["gomod"]
+  # The name of the package
+  packages_names: "github.com/release-engineering/retrodep/v2"
```

Comment:
If a single value is expected, let's rename it to `package_name`. Otherwise, turn it into a list.

Reply:
Since data["packages"] is a list, I'll turn it into a list.